packages/client/src/components/SafeOwners.js

```diff
@@ -63,13 +74,23 @@ function SafeOwners({ address, safeOwners, setSafeOwners }) {
               Owner Address<span className="has-text-red">*</span>
             </label>
             <div className="is-flex">
-              <input
-                className="p-4 rounded-sm flex-1"
-                type="text"
-                placeholder="Enter user's FLOW address"
-                value={so?.address}
-                onChange={(e) => onOwnerAddressChange(e.target.value, idx + 1)}
-              />
+              <div className="flex-1" style={{ position: "relative" }}>
+                <input
+                  className="p-4 rounded-sm"
+                  style={{ width: "100%" }}
```

Comment:
could use `column is-full` in className also

Reply:
good to know, updated all instances to do this instead of the hardcoded styles